Scoop/View/EmptyStateView.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  EmptyStateView.swift
+//  Scoop
+//
+//  Created by Caitlyn Jin on 10/28/23.
+//
+
+import Foundation
+import UIKit
+
+class EmptyStateView: UIView {
+
+    // MARK: - Views
+
+    private let imageView = UIImageView()
+    private let titleLabel = UILabel()
+    private let subLabel = UILabel()
+
+    // MARK: - Set Up Views
+
+    func setup(image: UIImage, title: String, subtitle: String) {
+        imageView.image = image
+        titleLabel.text = title
+        subLabel.text = subtitle
+
+        setupImageView()
+        setupTitleLabel()
+        setupSubLabel()
+    }
+
+    private func setupImageView() {
+        imageView.contentMode = .scaleAspectFit
+        imageView.layer.masksToBounds = true
+        addSubview(imageView)
+
+        imageView.snp.makeConstraints { make in
+            make.centerX.equalToSuperview()
+            make.top.equalToSuperview().inset(340)
```

Comment:
Is it alright for these two constraints to be constants? Not sure if 340 inset would look weird for something like an iPhone SE. Just a suggestion though!

Reply:
Changed this!